plugins/sql.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: sql
+spec:
+  version: v0.2.11
+  homepage: https://github.com/yaacov/kubectl-sql
+  platforms:
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/yaacov/kubectl-sql/releases/download/v0.2.11/kubectl-sql.tar.gz
+    sha256: 34b42e057cb7bf4d09340469d18ea7ecaca6f109d893a3fc5782dad84c8a0c94
+    files:
+    - from: "*"
+      to: "."
+    bin: kubectl-sql
+  shortDescription: Use SQL like language to query the Kubernetes cluster manager.
```

It's most important to be very concise here, because the short description will be truncated at ~50 chars.
```suggestion
  shortDescription: Query the cluster via pseudo-SQL
```